userguide/content/en/docs/adding-content/search.md

```diff
@@ -141,20 +141,27 @@ algolia_docsearch: true
    * In `head-end.html` add the DocSearch CSS:
 
       ```html
-      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" />
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" integrity="sha512-O6ywtjtiV4QGgC6cD75C2Tf04SlTn9vvka8oV/dYpDL1JWM4lVP0QoZbdZt5RLtOWDpaP+MObzUrwl43ssqfvg==" crossorigin="anonymous" />
       ```
 
-
-   * In `body-end.html` add the DocSearch script, replacing the `docsearch` details with the snippet you get from Algolia (the example below is Algolia's own site index!). You must provide `#docsearch` as your `container` value as that's the ID of the `div` we provide in Docsy's layout:
+   * In `body-end.html` add the DocSearch script, replacing the `docsearch` details with the snippet you get from Algolia (the example below is Algolia's own site index!). You must provide two `docsearch` functions, with their `container` values matching the IDs of the `div` elements we provide in Docsy's layout:
```

Comment:
```suggestion
   * In `body-end.html`, add the DocSearch script. In the code below, replace `docsearch()` argument values for `appId`, `apiKey`, and `indexName` with those provided by Algolia. Use the `container` values shown, since they correspond to the IDs found in Docsy's layout:
```

Reply:
Changed wording as proposed.